pkg/apis/crd/v1alpha2/types.go

```diff
@@ -338,7 +338,14 @@ type SubnetIPRange struct {
 
 type IPPoolStatus struct {
 	IPAddresses []IPAddressState `json:"ipAddresses,omitempty"`
-	// TODO: add usage statistics
+	Usage       IPPoolUsage      `json:"usage,omitempty"`
+}
+
+type IPPoolUsage struct {
```

Comment:
Do you think we can combine this type with `ExternalIPPoolUsage`?

Reply:
Technically this is doable of course. IDK enough the external pool functionality to conclude whether these two feature will have the same counters forever.
@tnqn do you have an opinion?

Reply:
I think it's a good suggestion. They should be same from pool's perspective, I can't think of any difference between the two cases for now.
I'm fine with making `ExternalIPPoolStatus` use `IPPoolUsage` if it doesn't impact API compatibility.